packages/kit/src/views/Prime/pages/PagePrimeTransfer/components/PrimeTransferDirection.tsx

```
@@ -0,0 +1,514 @@
+    throw new OneKeyLocalError('Random number is required');
+  }
+  if (randomNumber.length !== 6) {
+    throw new OneKeyLocalError('Random number is incorrect');
+  }
+  // userPart:     338713
+  // randomNumber: 576123
+  // verifyCode:   804836
+  const verifyCode = userPart
+    .split('')
+    .map((digit, index) => {
+      const userDigit = parseInt(digit, 10);
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)